test/screenplay/places/builder_test.exs

```diff
@@ -0,0 +1,307 @@
+defmodule Screenplay.Places.BuilderTest do
+  use ExUnit.Case
+
+  import Mox
+
+  alias Screenplay.Places.Cache, as: PlacesCache
+  alias Screenplay.Places.{Builder, Place}
+  alias Screenplay.ScreensConfig.Cache, as: ScreensConfigCache
+
+  setup_all do
+    _ = start_supervised(Screenplay.Places.Cache)
+    :ok
+  end
+
+  describe "handle_info/2" do
+    setup do
+      seed_screens_config_cache()
+
+      on_exit(fn ->
+        PlacesCache.delete_all()
+        ScreensConfigCache.delete_all()
+      end)
+
+      %{state: %{stops_mod: Screenplay.Stops.Mock, routes_mod: Screenplay.Routes.Mock}}
+    end
+
+    test "adds all parent stations as places", %{state: state} do
+      expect(Screenplay.Stops.Mock, :fetch_parent_stops, 2, fn _stop_ids ->
+        []
+      end)
```

Comment:
nitpick: Some of these empty `expect`s seem like they should fit on one line, which could improve readability.